### PR TITLE
Rishfilet/accessibilty buttons

### PR DESCRIPTION
**Pull request by rishFilet**
Two things were done here:
1. Made the elements and image in the first section responsive. I had to to do it for each element. If this does not work or look good on your screens, ill look into making a container and scaling them like that. 
2. Removed the border for the nav buttons whcih appears after pressing tab

This was updated with master.

### CODE REVIEW

File: src/components/Navbar/NavTabs.js [hunk 1]
```diff
@@ -113,6 +115,26 @@ function NavTabs() {
     ))
   }
 
+
+  function handleFirstTab(e) {
+    if (e.keyCode === 9) {
+      document.body.classList.add("user-is-tabbing")
+
+      window.removeEventListener("keydown", handleFirstTab)
```
Comment (hanlinc27) [inline src/components/Navbar/NavTabs.js:123]: image

- this causes the build to fail, can you implement the fix that @gh0stl0nely did before for this issue?

**Review by hanlinc27 — CHANGES_REQUESTED** [review]

File: src/components/Navbar/NavTabs.js (hunk 1 above)
Comment (rishFilet) [reply src/components/Navbar/NavTabs.js:123]: Fixed by using useEffects as suggested by Khoi

**Review by airshiprook — APPROVED** [review]
I would suggest removing the code that removes focus. 
See this link for more info: https://developers.google.com/web/fundamentals/accessibility/focus.
Other wise everything looks great on my MAC. I approve to have this merged.

**Comment by rishFilet** [comment]
> I would suggest removing the code that removes focus.
> See this link for more info: https://developers.google.com/web/fundamentals/accessibility/focus.
> Other wise everything looks great on my MAC. I approve to have this merged.

What do you mean remove the code that removes focus?

**Comment by airshiprook** [comment]
> > I would suggest removing the code that removes focus.
> > See this link for more info: https://developers.google.com/web/fundamentals/accessibility/focus.
> > Other wise everything looks great on my MAC. I approve to have this merged.
> 
> What do you mean remove the code that removes focus?

This file "src/components/Navbar/NavAccessibility.css" is removing focus right? I was saying that we should consider removing it as Focus is useful for users navigating the site via TAB/screen-reader.

**Comment by rishFilet** [comment]
> > > I would suggest removing the code that removes focus.
> > > See this link for more info: https://developers.google.com/web/fundamentals/accessibility/focus.
> > > Other wise everything looks great on my MAC. I approve to have this merged.
> > 
> > 
> > What do you mean remove the code that removes focus?
> 
> This file "src/components/Navbar/NavAccessibility.css" is removing focus right? I was saying that we should consider removing it as Focus is useful for users navigating the site via TAB/screen-reader.

So essentially that's what the change was, if you go to NavTabs.js i added a function in there that calls on this specific class to remove focus. It does not remove focus entirely. If you try it out by tabbing you will there is still focus but previously the focus would not disappear (when you use the mouse) until you refresh the page. Now it removes the focus when the user is not tabbing

**Comment by airshiprook** [comment]
> > > I would suggest removing the code that removes focus.
> > > See this link for more info: https://developers.google.com/web/fundamentals/accessibility/focus.
> > > Other wise everything looks great on my MAC. I approve to have this merged.
> > 
> > 
> > What do you mean remove the code that removes focus?
> 
> This file "src/components/Navbar/NavAccessibility.css" is removing focus right? I was saying that we should consider removing it as Focus is useful for users navigating the site via TAB/screen-reader.

Wait I think I misunderstood this. This is for the underline right? Anyhow disregard the comment. I think we are good as is.

**Comment by airshiprook** [comment]
> > > > I would suggest removing the code that removes focus.
> > > > See this link for more info: https://developers.google.com/web/fundamentals/accessibility/focus.
> > > > Other wise everything looks great on my MAC. I approve to have this merged.
> > > 
> > > 
> > > What do you mean remove the code that removes focus?
> > 
> > 
> > This file "src/components/Navbar/NavAccessibility.css" is removing focus right? I was saying that we should consider removing it as Focus is useful for users navigating the site via TAB/screen-reader.
> 
> So essentially that's what the change was, if you go to NavTabs.js i added a function in there that calls on this specific class to remove focus. It does not remove focus entirely. If you try it out by tabbing you will there is still focus but previously the focus would not disappear (when you use the mouse) until you refresh the page. Now it removes the focus when the user is not tabbing

Gotcha, thanks for explaining that.

**Comment by dghazi12** [comment]
The tablet and mobile view still seem slightly off on my screen. Disregarding the navbar I am seeing the following: 

Mobile View

Tablet View

**Review by dghazi12 — APPROVED** [review]

**Comment by BBoutet1** [comment]
@dghazi12 we are still working on the navbar responsiveness. Hamburger menu not started. If the rest is ok, you can approve and merge.

**Comment by BBoutet1** [comment]
@rishFilet the Donate Button is no more vertically centered and aligned with tabs.

**Review by hanlinc27 — APPROVED** [review]

**Pull request merged**